jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,605 @@
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "\n",
+    "\n",
+    "    }\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print the response object\n",
+    "---\n",
+    "Retrieve the response object as text to print out if there are no errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ORDERS_API_URL = 'https://api.planet.com/compute/ops/orders/v2'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "param_req = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "    order_params), auth=auth, headers={'content-type': 'application/json'})\n",
+    "\n",
+    "print(param_req.text)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Place the order\n",
+    "---\n",
+    "The following code places the order."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def place_order(order_params, auth):\n",
+    "    response = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "        order_params), auth=auth, headers=headers)\n",
+    "    order_id = response.json()['id']\n",
+    "    order_url = ORDERS_API_URL + '/' + order_id\n",
+    "    return order_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_url = place_order(order_params, session.auth)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Poll for data success\n",
+    "---\n",
+    "When you're ready to pull the data, you can poll for when it's ready. Here, the polling is limited to a number of loops routine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def poll_for_success(order_url, auth, num_loops=30):\n",
+    "    count = 0\n",
+    "    while(count < num_loops):\n",
+    "        count += 1\n",
+    "        r = requests.get(order_url, auth=session.auth)\n",
+    "        response = r.json()\n",
+    "        state = response['state']\n",
+    "        print(state)\n",
+    "        end_states = ['success', 'failed', 'partial']\n",
+    "        if state in end_states:\n",
+    "            break\n",
+    "        time.sleep(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "poll_for_success(order_url, auth)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r = requests.get(order_url, auth=session.auth)\n",
+    "response = r.json()\n",
+    "results = response['_links']['results']\n",
+    "\n",
+    "print([r['name'] for r in results])\n"
```

Comment:
Printing out in a single line is very readable, try something like this: `print("\n".join([r['name'] for r in results]))`